README.md

```diff
@@ -9,4 +10,57 @@ This project is to integrate CO2 emission intensity data into KEDA scaling decis
 - Electricity Maps https://static.electricitymaps.com/api/docs/index.html
 
 ## Deployment Scheme
-![Deployment](/images/deployment.png)
\ No newline at end of file
+![Deployment](/images/deployment.png)
+Relates to https://github.com/kedacore/keda/issues/3467
+
+## Target User Experience:
+
+- Admin can enable Carbon awareness for kubernetes resources, without requiring applications code change.
+- Admin defines Carbon Intensity thresholds to scale up/down based on, and AllowedMaxReplicaCount as a scaling target for the Scaled Object (Deployment or StatefulSet).
+
+``` yaml
```

Comment:
This section is confusing as this is not how it works today; we should maybe clarify that more here?